packages/rum/src/viewCollection.ts

```diff
@@ -37,19 +37,22 @@ export function startViewCollection(location: Location, lifeCycle: LifeCycle, se
   trackHistory(() => {
     if (areDifferentViews(currentLocation, location)) {
       currentLocation = { ...location }
+      currentView.triggerUpdate()
       currentView.end()
       currentView = newView(lifeCycle, currentLocation, session)
     }
   })
 
   // Renew view on session renewal
   lifeCycle.subscribe(LifeCycleEventType.SESSION_RENEWED, () => {
+    // do not trigger view update to avoid wrong data
```

Comment:
Instead of introducing a `triggerUpdate` function, I would rather introduce a `endWithoutUpdate` (or add an option `end({ update: false })` or something like that). This makes things clearer : `startViewCollection` is only responsible of creating and ending views, and `newView` is responsible to update the view during its lifecycle.

Reply:
both could work, I find it more clear to separate the end from the update instead of having two ways to end a view. For me the two concerns are separated since there is other conditions where we want to trigger an update, cf keep alive.